lib/auth-handler.js

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Honestly, that is all I ever see. `browserify`, `mocha`, `yoeman`, `karma`, `npm` and `Node.js` all follow this pattern.

I only prefer that because it is what I am used too. I am happy to change to camelCase.